### DISCUSSION TITLE
Netbox demo site internal server error

### DISCUSSION THREAD

**Discussion by thegreekman** [netbox-community/netbox#13141] [General]
When I try to access https://demo.netbox.dev I get an internal server error. Anyone else getting this?

**Comment by abhi1693** [comment]
This is due to https://github.com/netbox-community/netbox/issues/13116. Once a new version is released today, the site will be back online.

**Reply by thegreekman** [reply]
Thanks

**Reply by jeremystretch** [reply]
The demo is back up now on v3.5.6.

**Reply by thegreekman** [reply]
Thank you
